terminal.py

```diff
@@ -95,6 +95,10 @@ def main():
     menu_parser.add_argument("opt", choices=choices)
     completer = NestedCompleter.from_nested_dict({c: None for c in choices})
 
+    if os.name == "nt":
```

Comment:
@aia a discord user had his windows failing here, see:

<img width="699" alt="Captura de ecrã 2021-04-23, às 21 41 36" src="https://user-images.githubusercontent.com/25267873/115927678-b5ec6300-a47c-11eb-8b6e-d61d4aa8c2d2.png">

Reply:
That's probably an issue with the terminal that they are using.

I don't have any issues on Windows 10 and CI does not have any issues either.